src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -1436,7 +1436,7 @@ describe('ReactDOMComponent', () => {
 
     it('should warn about class', () => {
       spyOn(console, 'error');
-      ReactDOMServer.renderToString(React.createElement('div', {class: 'muffins'}));
+      ReactTestUtils.renderIntoDocument(React.createElement('div', {class: 'muffins'}));
```

Comment:
I suppose we should ideally have all our warnings replicated on the server and therefore have test for each one in a server-environment. Unless it's for updates etc.